mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -20,97 +41,115 @@ CLASS net/minecraft/unmapped/C_xmldumst net/minecraft/world/GameRules
 		ARG 1 key
-		ARG 2 rules
+		ARG 2 other
 		ARG 3 server
 	METHOD m_tahqotkb (Ljava/util/Map$Entry;)Lnet/minecraft/unmapped/C_xmldumst$C_ubszziqk;
-		ARG 0 e
-	METHOD m_vkgvfrzn getBoolean (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;)Z
-		ARG 1 rule
+		ARG 0 entry
+	METHOD m_vkgvfrzn getBooleanGameRuleValue (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;)Z
+		ARG 1 gameRule
 	METHOD m_vwilyacl (Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/unmapped/C_xmldumst$C_juzzykai;)V
 		ARG 0 server
 		ARG 1 rule
 	METHOD m_wqjrkanc accept (Lnet/minecraft/unmapped/C_xmldumst$C_avotqrag;Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
 		ARG 0 consumer
 		ARG 1 key
 		ARG 2 type
+	METHOD m_zerrhast (Ljava/util/Map$Entry;)Lnet/minecraft/unmapped/C_xmldumst$C_ubszziqk;
+		ARG 0 entry
 	METHOD m_zuudbppt register (Ljava/lang/String;Lnet/minecraft/unmapped/C_xmldumst$C_vkqoqsmj;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
 		ARG 0 name
 		ARG 1 category
 		ARG 2 type
 	CLASS C_avotqrag Visitor
 		COMMENT A visitor used to visit all game rules.
-		METHOD m_hsodkmym visitBoolean (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
-			COMMENT Visit a boolean rule.
+		METHOD m_hsodkmym visitBooleanGameRule (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+			COMMENT Visit a boolean game rule.
 			COMMENT
-			COMMENT <p>Note {@link #visit(GameRules.Key, GameRules.Type)} will be called before this method.
+			COMMENT <p>Note {@link #visitGameRule(GameRules.Key, GameRules.Type)} will be called before this method.
 			ARG 1 key
 			ARG 2 type
-		METHOD m_mfrkzoox visit (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+		METHOD m_mfrkzoox visitGameRule (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
 			ARG 1 key
 			ARG 2 type
-		METHOD m_vtezodau visitInt (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
-			COMMENT Visit an integer rule.
+		METHOD m_vtezodau visitIntGameRule (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+			COMMENT Visit an integer game rule.
 			COMMENT
-			COMMENT <p>Note {@link #visit(GameRules.Key, GameRules.Type)} will be called before this method.
+			COMMENT <p>Note {@link #visitGameRule(GameRules.Key, GameRules.Type)} will be called before this method.
 			ARG 1 key
 			ARG 2 type
 	CLASS C_bctfwntr Acceptor
-		METHOD call call (Lnet/minecraft/unmapped/C_xmldumst$C_avotqrag;Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+		METHOD call accept (Lnet/minecraft/unmapped/C_xmldumst$C_avotqrag;Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
```

Comment:
```suggestion
		METHOD call (Lnet/minecraft/unmapped/C_xmldumst$C_avotqrag;Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
```
we like to leave names whose obf name isn't proguarded the same as mojang names them. a plugin will make sure that the stats still consider this mapped!